web/src/main/kotlin/com/example/ajaxproject/repository/cacheable/CacheablePrivateRoomRepository.kt

```diff
@@ -0,0 +1,36 @@
+package com.example.ajaxproject.repository.cacheable
+
+import com.example.ajaxproject.model.PrivateChatRoom
+import com.example.ajaxproject.repository.PrivateChatRoomRepository
+import com.example.ajaxproject.repository.redis.PrivateChatRoomRedisRepository
+import org.springframework.beans.factory.annotation.Qualifier
+import org.springframework.stereotype.Repository
+import reactor.core.publisher.Mono
+
+@Repository
+class CacheablePrivateRoomRepository<T>(
```

Comment:
Why you have parameterized the repository to `<T>` ?

Reply:
discussed it